### ISSUE TITLE
Not Working on 1.17.41

### ISSUE THREAD

**Issue by ghost** [keowu/Minecraft-Windows-10-Trial-Bypass#1]
Hey there, it crashes on a GUI after "bypassing." Please Fix, thank you,

**Comment by keowu** [comment]
As soon as possible when i have an appropriate time i will check the changes in the new version and fix the memory offsets to make a new solution, possibly the new version contained invalid offsets causing the termination bug, will also be checked, thanks for your feedback reporting that.

**Comment by ghost** [comment]
Ok, thank you. By the way, do you just use CE or something to check the offsets? I would really like to start doing more memory-editing things like this :)

**Comment by keowu** [comment]
Sorry for the delay in the response, to check the offsets first I use a debugger and yes the CE is very good for that, after finding the correct offset I use a disassembler to analyze the region and only then consult the best patch, you can replace this when using only CE which is a great tool for fast memory analysis.

**Comment by griffi-gh** [comment]
any success?

**Comment by keowu** [comment]
Yes, I've worked on a fix and I'm testing it, I have to configure my Qt environment and in the next few weeks I'll do it, I'll update, I'll recompile and get it ready for download again, I'll leave a gif showing how it works, if anyone wants to contribute if they have the Qt already configured, I can move the new patch list to version **v1.18.2** and we'll be ready to go long before that, I plan to organize the repository by version from now on to better organize it
![minecraft_trial_bypass_1182](https://user-images.githubusercontent.com/42323126/145696716-9a3f8d94-76f2-4b1e-882a-592009183da1.gif)
.

**Comment by bluefoxy009** [comment]
@keowu If you want people to contribute you _could_ make a write-up on how to find the offsets/make the actual bypass, like another README type thing

**Comment by keowu** [comment]
Thanks for the suggestion, it looks good to me, as soon as we have some time we'll put it into practice, in the meantime I'll think about the best way to make a complete README that covers some necessary concepts.

**Comment by bluefoxy009** [comment]
Great Idea, I personally don’t know anything so I think that could be a possible idea.
